diff --git a/prebuilt/Android.mk b/prebuilt/Android.mk
index effa273..7805fcc 100755
--- a/prebuilt/Android.mk
+++ b/prebuilt/Android.mk
@@ -515,14 +515,16 @@ include $(BUILD_PREBUILT)
 
 ifeq ($(TW_INCLUDE_CRYPTO), true)
     ifneq ($(TW_CRYPTO_USE_SYSTEM_VOLD),)
-        # Prebuilt vdc_pie for pre-Pie SDK Platforms
-        include $(CLEAR_VARS)
-        LOCAL_MODULE := vdc_pie
-        LOCAL_MODULE_TAGS := optional
-        LOCAL_MODULE_CLASS := EXECUTABLES
-        LOCAL_MODULE_PATH := $(TARGET_RECOVERY_ROOT_OUT)/system/bin
-        LOCAL_SRC_FILES := vdc_pie-$(TARGET_ARCH)
-        include $(BUILD_PREBUILT)
+        ifneq ($(shell test $(PLATFORM_SDK_VERSION) -ge 28; echo $$?),0)
+            # Prebuilt vdc_pie for pre-Pie SDK Platforms
+            include $(CLEAR_VARS)
+            LOCAL_MODULE := vdc_pie
+            LOCAL_MODULE_TAGS := optional
+            LOCAL_MODULE_CLASS := EXECUTABLES
+            LOCAL_MODULE_PATH := $(TARGET_RECOVERY_ROOT_OUT)/system/bin
+            LOCAL_SRC_FILES := vdc_pie-$(TARGET_ARCH)
+            include $(BUILD_PREBUILT)
+        endif
     endif
 endif
 
